File: modul 2/PRAK204_2410817120001_Putri_Fatima_Azhara.c
#include <stdio.h>
#include <math.h>

int main ()
{
    float jari_jari, tinggi, volume, luas, keliling;

    scanf("%f %f", &jari_jari, &tinggi);

    volume = 22.0 / 7.0 * jari_jari * jari_jari * tinggi;
    luas = 2 * 22.0 / 7.0 * jari_jari * (jari_jari + tinggi);
    keliling = 2 * 22.0 / 7.0 * jari_jari;

    printf("\n");
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n\n", keliling);
    return 0;
}

File: modul 2/PRAK204_2410817120001_Putri_Fatima_Azhara.py
import math
angka = []

inputan = input().split()

angka.extend(inputan)

while(len(angka) < 2):
    inputan = input().split()
    angka.extend(inputan)

jari_jari = float(angka[0])
tinggi = float(angka[1])

volume = 22.0 / 7.0 * jari_jari * jari_jari * tinggi
luas = 2 * 22.0 / 7.0 * jari_jari * (jari_jari + tinggi)
keliling = 2 * 22.0 / 7.0 * jari_jari

print(" ")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}\n")